exercises/harry-potter/description.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
@petertseng help me out here.  This file doesn't appear [here](https://github.com/rpottsoh/x-common/tree/rpottsoh-patch-2/exercises/book-store).  I clearly see the empty file in the PR, but do not know how to remove it other than to close the PR and open a new one.